cmd/main.go

```diff
@@ -188,6 +189,8 @@ func main() {
 	ocmClient := getOCMClient(log, metricsManager)
 
 	Options.InstructionConfig.ReleaseImageMirror = Options.ReleaseImageMirror
+	Options.InstructionConfig.CheckClusterVersion = Options.CheckClusterVersion
+	Options.OperatorsConfig.CheckClusterVersion = Options.CheckClusterVersion
```

Comment:
Would it be easier to set up `envconfig` on the `InstructionConfig` and `OperatorsConfig` structs individually or are we doing this so we have the env var in just one place?

Reply:
I prefer that the env var would be in one place only